ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/WriteService.scala

```diff
@@ -87,14 +87,7 @@ trait WriteService
     *                        associated [[ReadService]].
     * @param transactionMeta : the meta-data accessible to all consumers of the
     *                        transaction. See [[TransactionMeta]] for more information.
-    * @param transaction     : the submitted transaction. This transaction can
-    *                        contain contract-ids that are relative to this transaction itself.
-    *                        These are used to refer to contracts created in the transaction
-    *                        itself. The participant state implementation is expected to convert
-    *                        these into absolute contract-ids that are guaranteed to be unique.
-    *                        This typically happens after a transaction has been assigned a
-    *                        globally unique id, as then the contract-ids can be derived from that
-    *                        transaction id.
+    * @param transaction     : the submitted transaction.
```

Comment:
I guess this is a proactive removal of the comment, but doesn't really fit this PR.

Reply:
I put back the comment